src/components/field-types/CodeEditorFieldComponent.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <validate :state="state" :custom="{'validate': isValid}">
+    <div class="form-group">
+      <label :for="field.id">{{ field.label }}</label>
+      <div id="styling" :style="style">
```

Comment:
1) Dont use ID's if you are not using them via css or tests.
2) try to use classes for elements which can be rendered multiple times i.e. two HTML fields will render two divs with the id 'styling' which will confuse possible tests